TableViewMatchers.containsRow method behaviour

Hello there,

I've started to use your great library to test my small application and currently facing weird behaviour of the TableViewMatchers.containsRow method. As name applies, it should return true if TableView has entry which matches to the row, entered as Object... parameter.
For example, I'm expecting true when TableView contains 
```
row 1 -> "Blue", 1, false
row 2 -> "Red", 4, true
```
and I'm looking for row to have values -> "Blue", 1, false.
However method returns true, as it expects both row to be looked for and order is compulsory.
Is it how it should behave? Then the name is misleading :(

Sure thing

This is the test case for my actual application (not the example from the initial message, but it's same idea)
    
 ```
   @Override
    public void start(Stage stage) {
        User client = new Client("t", "t");
        Basket userBasket = new Basket();

        Inventory inventory = new Inventory();
        try {
            inventory.addProducts(new Product("chicken", 1, 2.3),3);
            inventory.addProducts(new Product("apple", 0.151, 0.8), 2);
        } catch (InventoryException e) {
            e.printStackTrace();
        }

        client.setBasket(userBasket);

        stage.setScene(new Scene(InventoryScene.createInventoryTableView(inventory, client), 600, 400));
        stage.show();
    }

    @Test
    public void should_contain_data_in_rows_for_user() {
        verifyThat(".table-view", TableViewMatchers.containsRow("[apple, 0.151, 0.8, 2, false]"));
    }
```

Here I'm trying to test, if one of the rows in table view contains row with described values 
And, from `start` method, it's clear that row I'm looking for exists in table view.

However I'm getting following error 

```
java.lang.AssertionError: 
Expected: TableView has row: [[apple, 0.151, 0.8, 2, false]]
     but: was [[apple, 0.151, 0.8, 2, false], [chicken, 1.0, 2.3, 3, false]]
Expected :TableView has row: [[apple, 0.151, 0.8, 2, false]]
     
Actual   :[[apple, 0.151, 0.8, 2, false], [chicken, 1.0, 2.3, 3, false]]
```
So it seems like `TableViewMatchers.containsRow` method gets all cells and compares the cells I want against all cells from table view. Also, even if I pass both rows to this method as 
`TableViewMatchers.containsRow("[apple, 0.151, 0.8, 2, false], [chicken, 1.0, 2.3, 3, false]")`
it fails sometimes because order in which those rows appear in table view differs sometimes -> order in which I pass those parameters are important for the `TableViewMatchers.containsRow` method. So it seems like either error in the method itself or just name that misleading.

--- a/subprojects/testfx-core/src/main/java/org/testfx/matcher/control/TableViewMatchers.java
+++ b/subprojects/testfx-core/src/main/java/org/testfx/matcher/control/TableViewMatchers.java
@@ -87,8 +87,29 @@ public class TableViewMatchers {
     }
 
     /**
-     * Creates a matcher that matches all {@link TableView}s that have a row at the given {@code index} with
-     * all of the given cells.
+     * Creates a matcher that matches all {@link TableView}s that have a row at the given {@code index} that
+     * contains the given calues for each column of a {@code TableView}.
+     * <p>
+     * For example, given a {@code TableView} that has three columns:
+     * <pre>{@code
+     * TableColumn<RegularPolygon, String> nameColumn = new TableColumn<>("Name");
+     * TableColumn<RegularPolygon, Integer> numSidesColumn = new TableColumn<>("Number of Sides");
+     * TableColumn<RegularPolygon, Double> unitAreaColumn = new TableColumn<>("Area when Side = 1");
+     * polygonsTable.getColumns().setAll(nameColumn, numSidesColumn, unitAreaColumn);
+     * }</pre>
+     * Then to verify that such a {@code TableView}, contains, at index 3, a row for a {@code RegularPolygon}
+     * that has the name {@literal "Pentagon"}, the number of sides {@literal 5}, and a unit area of
+     * {@literal 1.720477401} one would use:
+     * <pre>{@code
+     * verifyThat("#polygonsTable", TableViewMatchers.containsRowAtIndex(3, "Pentagon", 5, 1.720477401);
+     * }</pre>
+     * Where the types of each argument, after the row index, correspond to the types of the {@code TableColumn}s
+     * which in our example is {@code (String, Integer, Double)}.
+     *
+     * @param rowIndex the row number (starting from 0) that must contains the given cell values
+     * @param cells The values or String representations of the values (e.g. the result of calling {@code toString()})
+     * contained in the row at a given index you want to verify a {@code TableView} contains - one such value for
+     * each column of that {@code TableView}.
      */
     @Factory
     public static Matcher<TableView> containsRowAtIndex(int rowIndex, Object... cells) {
@@ -108,7 +129,28 @@ public class TableViewMatchers {
     }
 
     /**
-     * Creates a matcher that matches all {@link TableView}s that have a row containing all the given cells.
+     * Creates a matcher that matches all {@link TableView}s that have at least one row that contains the given
+     * values for each column of a {@code TableView}.
+     * <p>
+     * For example, given a {@code TableView} that has three columns:
+     * <pre>{@code
+     * TableColumn<Person, String> nameColumn = new TableColumn<>("Name");
+     * TableColumn<Person, Double> bmiColumn = new TableColumn<>("Body Mass Index");
+     * TableColumn<Person, Boolean> membershipColumn = new TableColumn<>("Gym Membership Valid");
+     * fitnessTable.getColumns().setAll(nameColumn, bmiColumn, membershipColumn);
+     * }</pre>
+     * Then to verify that such a {@code TableView}, contains at least one row with a {@code Person}
+     * that has the name {@literal "Dan Anderson"}, the body mass index {@literal 28.83}, and a valid
+     * gym membership ({@literal true}) one would use:
+     * <pre>{@code
+     * verifyThat("#fitnessTable", TableViewMatchers.containsRow("Dan Anderson", 28.83, true);
+     * }</pre>
+     * Where the types of each argument correspond to the types of the {@code TableColumn}s which
+     * in our example is {@code (String, Double, Boolean)}.
+     *
+     * @param cells The values or String representations of the values (e.g. the result of calling {@code toString()})
+     * contained in the row you want to verify a {@code TableView} contains - one such value for each column of
+     * that {@code TableView}.
      */
     @Factory
     public static Matcher<TableView> containsRow(Object...cells) {

